plugins/inputs/tacacs/tacacs.go

```diff
@@ -0,0 +1,150 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package tacacs
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"net"
+	"sync"
+	"time"
+
+	"github.com/nwaples/tacplus"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Tacacs struct {
+	Servers         []string        `toml:"servers"`
+	Username        config.Secret   `toml:"username"`
+	Password        config.Secret   `toml:"password"`
+	Secret          config.Secret   `toml:"secret"`
+	RemAddr         string          `toml:"request_ip"`
```

Comment:
Can we please use a more speaking name for the variable!?
```suggestion
	RequestAddr         string          `toml:"request_ip"`
```

Reply:
Initially it was called like that because RemAddr naming is used in the underlying tacacs client library dependency, and thats the variable it sets. But I agree RemAddr is from the viewpoint of the tacacs server, for which the remote address is the telegraf agent. From telegraf agent viewpoint, this is our request address, so I agree that this should be changed.

When creating the client object, the old variable name will still appear in the code though:

```
tacplus.AuthenStart{
		Action:        tacplus.AuthenActionLogin,
		AuthenType:    tacplus.AuthenTypeASCII,
		AuthenService: tacplus.AuthenServiceLogin,
		PrivLvl:       1,
		Port:          "heartbeat",
		RemAddr:       t.RequestAddr,
	}
```